### DISCUSSION TITLE
Hide/Show specific parts.

### DISCUSSION THREAD

**Discussion by CharlieG42** [OpenOrienteering/mapper#2161] [Questions & Answers] [answered]
Hi,

Is there a way to hide /show and print/not print specific parts ?

For exemple, in my project I have 3 parts : roads, woods and buildings.
For a training I would like print only the parts with the woods and the road.

For the moment I create a copy of my project and I delete the part that I don't want see... But maybe there is a feature to do that differently ?

Thanks in advance for your help.

Charlie.

**Comment by dl3sdo** [comment, marked as answer]
There is an issue (#1959) which addresses this topic but the code is not yet part of Mapper.

**Reply by lpechacek** [reply]
Traditionally (outside the Mapper world) this task is accomplished by **hiding the corresponding map symbols**. This can be done in Mapper too. 

The map part visibility controls mentioned by Matthias are under discussion, so don't hold your breath for it.

HTH
